src/main/proto/wfa/measurement/api/v2alpha/exchange_workflow.proto

```diff
@@ -37,15 +45,52 @@ message ExchangeWorkflow {
     // the step-specific name to a label for the input. The step is ready to run
     // when, for each input label, there is a completed step with the same
     // output label.
+    //
+    // Inputs to the protocol are provided via InputStep step types -- this
+    // ensures that all inputs to all steps appear as some output label.
     map<string, string> input_labels = 3;
     map<string, string> output_labels = 4;
 
-    // TODO(world-federation-of-advertisers/experimental#81): add step details.
-  }
+    // Compares the inputs to determine if they overlap sufficiently.
+    message IntersectAndValidateStep {
+      // The maximum number of items in to allow.
+      int64 max_size = 3;
 
-  enum Party {
-    PARTY_UNSPECIFIED = 0;
-    MODEL_PROVIDER = 1;
-    DATA_PROVIDER = 2;
+      // The minimum fraction of items that must exist in both input data sets.
+      float minimum_overlap = 4;
```

Comment:
which party specifies this?

Reply:
It'll be baked into the ExchangeWorkflow that they both agree on.